javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,10 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var newArray = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```

Comment:
Your function would make more sense (`filterOutStrings`) if you concentrated on eliminating the strings instead.

```js
if (typeof values[i] !== 'string') {
```

Reply:
<img width="775" alt="Screen Shot 2021-01-29 at 2 46 28 PM" src="https://user-images.githubusercontent.com/74885918/106335470-bf7d7a80-6241-11eb-8659-2b796486fded.png">